package.json

```diff
@@ -53,18 +53,21 @@
           "icon": "${bootable-icon}"
         }
       ],
-      "icons/imagesList": [
+      "icons/image": [
         {
           "when": "ostree.bootable in imageLabelKeys",
           "icon": "${bootable-icon}"
         }
       ],
-      "badges/imagesList": [
+      "badges/image": [
         {
           "when": "ostree.bootable in imageLabelKeys",
           "badge": {
-            "name": "bootc",
-            "style": "bg-green-600"
+            "label": "bootc",
+            "color": {
+              "dark": "#32CD32",
+              "light": "#FF0000"
```

Comment:
if you prefer I think you can choose a class name for the color instead of #rgb

Reply:
Doesn't look like you can with the background values as tailwind doesn't pre-render them if they are passed in dynamically (looks like you can only show it if somewhere in the code we have the colour code... 

I just tried with bg-green-300, but bg-sky-300 works. It's odd.

I've gone ahead and changed it to a light-green for now though!


https://github.com/containers/podman-desktop-extension-bootc/assets/6422176/50dc89d7-27c1-4d6d-865d-185edd42c89c

Reply:
It's not surprising that blue didn't work, since we have a restricted palette and blue is not in it:
https://github.com/containers/podman-desktop/blob/main/tailwind.config.cjs

No idea why green didn't work, but sky looks good. As Florent said it's probably better to go with that (or any other class name that works) over rgb.

Reply:
That was just a test :) but green doesn't work since we have it in our palette, but we do not call green anywhere, so tailwind doesn't "pre-render" it / bring it in when compiling. I can't find the issue, but I remember we had this problem before.

I know that "sky" works because we use that colour for other container providers.

I don't have a problem going with sky!